pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "nautobot-circuit-maintenance"
-version = "2.2.4"
+version = "2.3.0"
```

Comment:
bump to `2.3.1b1` perhaps?

Reply:
do you mean to prepare for the next release? I haven't seen this in the process, but it could be a good idea to consider